### PR TITLE
Add explanation on nightly builds.

### PR DESCRIPTION
**Pull request by wei2912**
Fixes the second issue of https://github.com/learnmath/lysa/issues/63.

### CODE REVIEW

**Comment by pharpend** [comment]
You need wayyyy more explanation - like explain what "building from source is", explain what plain-text markup is, etc. I will merge this, since it is an improvement. #63 is still not solved.

**Comment by wei2912** [comment]
In that case, don't merge it yet.

**Comment by pharpend** [comment]
Alright, then.

**Comment by wei2912** [comment]
I can't ELI5, so I'll pass the job to @beingbrown.

**Comment by pharpend** [comment]
Alright. I'll merge this and assign the issue to @beingbrown

**Comment by ghost** [comment]
Don't merge it; i'll ad wei as upstream and pull to him.

**Comment by pharpend** [comment]
Okay, then.

**Comment by ghost** [comment]
I'm closing this for now to clear the queue. We can resubmit when @wei2912 and I have something ironed out.

**Comment by pharpend** [comment]
@beingbrown let me get this straight, you're closing the issue because it is unsolved?

**Comment by ghost** [comment]
I'm closing the merge request. It won't merge because it conflicts, and we haven't solved the issues involved.
